src/DynamoCore/Models/PortModel.cs

```diff
@@ -80,6 +80,11 @@ public int LineIndex
             get { return portData.LineIndex; }
         }
 
+        /// <summary>
+        /// A flag indicating whether the port is considered connected.
+        /// </summary>
+        /// 
+        [Obsolete("Please use NodeModel.HasConnectedInput instead.")]
         public bool IsConnected
```

Comment:
I've obsoleted `IsConnected` because it is used in very few locations and is not as capable as checking `NodeModel.HasInput` or `NodeModel.HasConnectedInput`.